[IMP] Frontdesk: updating configurations (#12979)
odoo/documentation: content/applications/hr/attendances.rst

Attendances
===========

Odoo's **Attendances** application functions as a time clock. Employees are able to check in and out

> of work using a dedicated device in kiosk mode, while users are also able to check in and out of

Reviewer: can possibly link to the kiosk doc for readers who don't know what it is?

Author: Oooh yes- and the manual one as well!

work directly from the database. Managers can see who is available at any given time, create reports
to see everyone's hours, and gain insights on which employees are working overtime, or checking out
of work earlier than expected.

.. _attendances/access-rights:

Access rights
=============

It is important to understand how the different access rights affect what options and features users
can access in the **Attendances** application.

Every user in the database is able to check in and out directly from the database, without needing
access to the **Attendances** application. Additionally, all users can access their own attendance
records from their employee form in the **Employees** app.

Access to both the **Attendances** application, and the various features within the application is

> determined by access rights.

Reviewer: The original content is clear; I'm wondering if there are ways to tighten the structure of the explanation and emphasize the importance of understanding access levels, and communicating that in a more formalized documentation tone (because we take ourselves very seriously :sparkles:) 

This is a suggestion; feel free to ignore it if u don't like it! It's all personal preference (:

Understanding access rights is essential to navigating the Attendances application. While every user can check in and out directly from the database—and view their own attendance records via their employee form in the Employees app—access to the Attendances application itself, and the features within it, depends on assigned permissions.

:guilabel:`Attendances` field, the options are either to leave the field blank or select
:guilabel:`Administrator`.

If the :guilabel:`Administrator` option is selected, the user has full access to the entire
**Attendances** application, with no restrictions. They can view all employee attendance records,
enter *Kiosk Mode* from the application, access all reporting metrics, and make modifications to the
settings. If left blank, the user does **not** have access to the **Attendances** application.

.. note::
   If a user does **not** have :guilabel:`Administrator` rights for the **Attendances** app, they
   are **not** able to open the app, even though it appears on the main database dashboard. An
   :guilabel:`Access Error` pop-up message appears, stating:

   `You do not have enough rights to access the fields "attendance_manager_id" on Employee
   (hr.employee). Please contact your system administrator.`

   Users who cannot access the **Attendances** app can still :doc:`check in and check out
   <../hr/attendances/check_in_check_out>` of work within the database, using the :guilabel:`🔴 (red

>    circle)` or :guilabel:`🟢 (green circle)` that are always available at the top of the database.

Author: I left this as a note only because there is a full check-in and check-out doc, as well as a kiosk doc, which goes into super detail about checking in (they have both been linked in this doc as well, where applicable). So I left these instructions just so they were here for whoever ONLY reads this doc =D

.. _attendances/approvers:

Approvers
---------

The **only** other scenario where different information may be accessible in the **Attendances**
application is for approvers. If a user does *not* have administrative rights for the
**Attendances** application, but they are set as an employee's approver for the **Attendances**
application, that user is able to view the attendance records for that specific employee, as well as
make modifications to that employee's attendance records, if necessary. This applies to all
employees for whom the user is listed as the **Attendances** application approver. Approvers are
typically managers, though this is not required.

> To see who the attendance approver is for an employee, navigate to the :menuselection:`Employees

Reviewer: instead of seeing, which assumes the user has the setting already enabled, how about

"To set an attendances approver for an employee, go to [instructions you have there!]"

application` and click on the specific employee. Click on the :guilabel:`Work Information` tab,

> scroll to the :guilabel:`APPROVERS` section, and check the :guilabel:`Attendance` field. The person

Reviewer: ```suggestion
scroll to the :guilabel:`APPROVERS` section, and set the :guilabel:`Attendance` field to the approver. The person
```

selected is able to view that employees' attendance records, both on the **Attendances** application
dashboard as well as in the attendance reports, and make modifications to their records.

Configuration
=============

Few configurations are needed in the **Attendances** application. Determining how employees check in
and out, defining how the kiosks function, and determining how extra hours are computed are all set

> in the Configuration menu. Navigate to the :menuselection:`Attendances application -->

Reviewer: can possibly shorten application to app!

Configuration` to access the configuration menu.

.. note::
   Any configuration item with an :icon:`fa-building-o` :guilabel:`(building)` icon is a
   company-specific configuration. Items without an :icon:`fa-building-o` :guilabel:`(building)`
   icon apply to all companies within the database.

Modes
-----

> - :guilabel:`Attendances from Backend` :icon:`fa-building-o`: activate this selection to allow users

Reviewer: totally optional, but i think feature is more consistent to how the rest of the team calls these. But, until we have a team guideline on this, it is all just personal preference (:
```suggestion
- :guilabel:`Attendances from Backend` :icon:`fa-building-o`: activate this feature to allow users
```

if you accept this suggestion, remember to change it with the other features in this bulleted list!

  to check in and out directly from the Odoo database. If this is not activated, users must use a
  kiosk to check in and out of work.
- :guilabel:`Automatic Check-Out` :icon:`fa-building-o`: activate this selection to automatically

>   check-out employees according to their working schedule, after a buffer of time has passed.

Reviewer: ```suggestion
  check out employees according to their working schedule, after a buffer of time has passed.
```

- :guilabel:`Tolerance`: this field appears only when the :guilabel:`Automatic Check-Out`
  :icon:`fa-building-o` option is enabled. Enter the amount of time, in hours, that must elapse
  after an employee's working hours have ended, before they are automatically checked out.

.. example::
   A company activates the :guilabel:`Automatic Check-Out` :icon:`fa-building-o` option, and sets
   the :guilabel:`Tolerance` to `2.00` hours. An employee's working hours are from 9:00 AM to 5:00
   PM.

   An employee checks in to work at 9:00 AM, and forgets to check-out. At 7:00 PM, they are

>    automaticlaly checked-out according to the configurations entered.

Reviewer: ```suggestion
   automatically checked out.
```

>    automaticlaly checked-out according to the configurations entered.

Reviewer: Great example that helps me understand how it works! possible suggestion for brevity:

With the :guilabel:`Automatic Check-Out` option enabled, and the :guilabel:`Tolerance` set to `2.00` hours, an employee checks in to work at 9:00 AM, and forgets to check-out at 5:00 PM. At 7:00 PM, they are automaticlaly checked out.
